<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PlotCanvas" name="canvasWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actExportCurrentSize"/>
   <addaction name="actExportSelectedSize"/>
  </widget>
  <action name="actExportCurrentSize">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export current size</string>
   </property>
   <property name="toolTip">
    <string>Export current size</string>
   </property>
  </action>
  <action name="actExportSelectedSize">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export selected size</string>
   </property>
   <property name="toolTip">
    <string>Export selected size</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotCanvas</class>
   <extends>QWidget</extends>
   <header>gpxviewer.plotcanvas</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actExportCurrentSize</sender>
   <signal>triggered()</signal>
   <receiver>PlotWindow</receiver>
   <slot>saveCurrentSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actExportSelectedSize</sender>
   <signal>triggered()</signal>
   <receiver>PlotWindow</receiver>
   <slot>saveSelectedSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveCurrentSize()</slot>
  <slot>saveSelectedSize()</slot>
 </slots>
</ui>
